### ISSUE TITLE
Improved error/warning messaging for CST appeals

### ISSUE THREAD

**Issue by annaswims** [department-of-veterans-affairs/va.gov-team#30030]
When https://api.va.gov/v0/appeals responds with 404 we should present a better error message. 
The 404 response means the veteran was not found in caseflow, meaning that we did not find any appeals for the veteran. It's not that the system is having trouble and it's unlikely to change in the next hour. 

![Screenshot from 2021-09-16 07-51-59](https://user-images.githubusercontent.com/19188/133640360-9bff3094-e4ee-403f-a31b-adfd0dae5e25.png)

**Comment by saderagsdale** [comment]
@saderagsdale Talk to Jacob about moving to squad 1.

**Comment by saderagsdale** [comment]
@thehastingsp bring to joe's attention.

**Comment by saderagsdale** [comment]
@jacobworrell moving this to squad 1.

**Comment by jsokohl** [comment]
"We're having trouble finding this [any] appeal[s]. You can still view your claims information, but you need to contact [whoever they need to contact]."

Something like this, perhaps. It seems that they have to take their query off line and into the meatspace world, not the digital one.

**Comment by skylerschain** [comment]
Looking at this again, isn't the "You do not have any submitted claims" message sufficient? Why do we even need an extra alert? There is not really an error here, or any action for the user to take. They just don't have any claims or appeals, because they haven't submitted any. If that's the case, the blue banner seems to communicate all they need to know. 

WDYT @jsokohl?

**Comment by jsokohl** [comment]
IIUC, the original ticket indicated that the 404 being returned is inaccurate--there is no identified appeal (not claim--the second message is perhaps just that--separate). They have claims...that were rejected, and they then appealed.  The veteran's come back to the CST looking for status on that _appeal_, but we can't find the appeal. So that's why I said the VA is "having trouble finding the appeal." Good question, @skylerschain -- but i think we should determine whether there is any recourse for the veteran.

**Comment by skylerschain** [comment]
Yeah, I guess the question is: Is this an actual error in finding an appeal, or does this user just not have any appeals? 

If they're not expecting to see any appeals, we don't need to show the error message. If they are (i.e. if one exists but we just can't fetch it), we should show a message. 

Maybe @Mottie or @michelpmcdonald knows the answer to the above?

**Comment by Mottie** [comment]
The "status is unavailable" message is shown when there are _any_ fetch claim errors. It's a default message

https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/claims-status/reducers/claimsV2.js#L50
